Dockerfile

```diff
@@ -34,24 +34,44 @@ RUN bundle exec bootsnap precompile --gemfile
 
 # The following steps are no longer needed with importmap
 # COPY package.json package-lock.json ./
-# RUN npm install
 
 # Copy application code
 COPY . .
-#
+
```

Comment:
can you add some comments to understand what these two env variables are doing, it can become difficult to recall this later